### PR TITLE
Cleanup bug report template and add wallet connection information

### PR DESCRIPTION
**Pull request by Aerilym**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -65,14 +64,35 @@ body:
       description: What browser resolution does this issue occur on? You can find this easily using tools like https://mdigi.tools/browser-resolution/
       placeholder: eg. 1280 x 795
     validations:
-      required: true
+      required: false
```
Comment (yougotwill) [inline .github/ISSUE_TEMPLATE/bug_report.yml:67]: Why is this not required but OS is so?

**Review by yougotwill — COMMENTED** [review]
1 question

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Aerilym) [reply .github/ISSUE_TEMPLATE/bug_report.yml:67]: Well OS is trivial but people might struggle with getting the browser resolution, browser res is also not very useful except for very specific ui bugs

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

**Pull request merged**